src/lib/uni/README.md

```diff
@@ -0,0 +1,55 @@
+The `uni` entrypoint is a completely new approach to the Layout library. The emphasis here
+is on simplicity, with as little defined in the library as possible. The goal is to create
+a utility around the hardest part of layout management: creating and defining interchangeable
+builders and coordinating that with the media query system in the browser.
+
+This is a backwards-incompatible approach to the current version of Angular Flex Layout, but
+we believe that this approach is more ergonomic in nature, and is the right way of doing things
+moving forward. That being said, this is completely opt-in, with no actual plans to transition
+anywhere in the near- or long-term.
+
+To use this is quite simple:
+
+### Step 1: Import the new module
+```ts
+// app.module.ts
+
+@NgModule({
+  imports: [UnifiedModule.withDefaults()] // brings in default tags and breakpoints
+})
+export class AppModule {}
+```
+
+### Step 2: Use the new syntax
+```html
+<div ngl flexAlign="start">
+  <bp tag="xs" flexAlign="end"></bp>
+  <bp tag="md" flexAlign="center"></bp>
+</div>
+```
+
+Here, `ngl` stands for "Angular (ng) Layout (l)". The `bp` elements are optional if
+you don't plan to use breakpoints in your template. Conceptually, it's cleaner than
+cramming all of the breakpoints on one root element, and the `bp` elements don't
```

Comment:
`bp` is short, but I think that it lacks a little bit of expressiveness. It won't be readable to people who aren't familiar with this library's APIs.

Is `<breakpoint` overly verbose? It seems a lot more readable and expressive.


```html
<div ngLayout flexAlign="start">
  <breakpoint tag="xs" flexAlign="end"></breakpoint>
  <breakpoint tag="md" flexAlign="center"></breakpoint>
</div>
```